resources/lib/image_server.py

```diff
@@ -119,10 +118,9 @@ def build_image(path):
             log.debug("Loading image from : {0} {1} {2}", image_count, server, url_full_path)
 
             try:
-                conn = httplib.HTTPConnection(server)
-                conn.request("GET", url_full_path)
-                image_responce = conn.getresponse()
-                image_data = image_responce.read()
+
+                image_responce = requests.get(thumb_url)
+                image_data = image_responce.content
```

Comment:
```suggestion
                image_response = requests.get(thumb_url)
                image_data = image_response.content
```
spelling is important